packages/fxa-content-server/tests/server/routes/validate-email-domain.js

```diff
@@ -57,7 +58,21 @@ registerSuite('routes/validate-email-domain', {
           assert.isTrue(res.json.calledOnceWith({ result: 'MX' }));
         });
       },
-
+      'responds with result {result: "skip"} when there is a skip record}': () => {
+        const dns = {
+          ...ERROR_CODES,
+          //Promises always resolved so no need to give a Resolver
+        };
+        const validateEmailDomainRoute = require(dns);
+        const enabled = config.get('mxRecordValidation');
+        enabled.enabled = false;
+        const req = { query: { domain: 'skip.abc.xyz' } };
+        const res = { json: sinon.stub() };
+        const route = validateEmailDomainRoute();
```

Comment:
`validateEmailDomainRoute` should be the module you changed above; see line 32 for the `require` call.  Then when you call `validateEmailDomainRoute()`, pass in teh config object.  You can mock that config object with sinon: `{get: sinon.stub().returns(false)}`.

Reply:
@chenba rather we can use this 
const stubConfigGet= sinon.stub(config,get)
stubConfigGet.withArgs('mxRecordValidate.enabled').returns(false)
stubConfigGet.restore()